### DISCUSSION TITLE
[Docs] Where is command to install only via composer?

### DISCUSSION THREAD

**Discussion by ampmonteiro** [laravel/framework#53312] [General]
Hi, 
after i listen to the interview of Creator of Laravel did to Youtuber ThePrimegen and another Youtuber Theo - t3.gg, mention about using `php.new`, i notice in the docs > installation section, that like v10:

```bash
composer create-project laravel/laravel:^10.0 example-app
```

in version 11, this command do not show in the docs, Why ?

It is fine to show or to exist several ways to install, however i think the traditional way,  should always be in Docs, 
so the `composer way to install / create` a project  should be mention in the docs.

Thanks.

**Comment by felipesantos2** [comment]
I think they're embracing their own tools, herd ...

Now in laravel 12 they've removed the direct install for docker from the docs.

**Comment by shaedrich** [comment]
Torchlight would offer a way to display composer, docker and herd install side by side like with PHPUnit and Pest in laravel/docs#9291
